ebpfapi/dllmain.cpp

```diff
@@ -8,7 +8,7 @@
 bool use_ebpf_store = true;
 
 BOOL APIENTRY
-DllMain(HMODULE hModule, DWORD ul_reason_for_call, LPVOID lpReserved)
+DllMain(HMODULE hModule, unsigned long ul_reason_for_call, void* lpReserved)
```

Comment:
for consistency with the rest of the code, could we use ```uintNN_t``` instead of language defined types (such as the ```unsigned long``` used here) instead?